Source/Details/ASLegacyCollectionLayoutCalculator.m

```diff
@@ -0,0 +1,44 @@
+//
+//  ASLegacyCollectionLayoutCalculator.m
+//  AsyncDisplayKit
+//
+//  Created by Huy Nguyen on 6/3/17.
+//  Copyright © 2017 Facebook. All rights reserved.
+//
+
+#import <AsyncDisplayKit/ASLegacyCollectionLayoutCalculator.h>
+
+#import <AsyncDisplayKit/ASCellNode.h>
+#import <AsyncDisplayKit/ASCollectionElement.h>
+#import <AsyncDisplayKit/ASDispatch.h>
+#import <AsyncDisplayKit/ASDisplayNode+Subclasses.h>
+#import <AsyncDisplayKit/ASElementMap.h>
+#import <AsyncDisplayKit/ASLayout.h>
+#import <AsyncDisplayKit/ASLayoutSpec.h>
+
+@implementation ASLegacyCollectionLayoutCalculator
```

Comment:
Instead of adding this, could we say "we have no calculator – we are in 'legacy mode'" and have the data controller go ahead and perform the eager allocation/measurement step as before in a separate codepath? It's a little confusing that this thing doesn't really calculate any layout.